src/seeders/20200106071439-demo-users.js

```diff
@@ -0,0 +1,45 @@
+import faker from 'faker';
+import BcryptService from '../services/bcrypt.service';
+
+const passwordHash = BcryptService.hashPassword('Pass1234');
+
+const demoUsers = {
+  up: (queryInterface, Sequelize) => queryInterface.bulkInsert('Users', [{
+    id: 31,
```

Comment:
The id should be auto-generated.

Reply:
Because I need to use the same Id in the preference seeders, 
Especially for line 5, 13, 17 If I remove the eslint rule and then try to assign variables, then the HTML won't work.

Reply:
Then you should look for a way to dynamically map each created user to their preference.